tools/jit/templates/aten_dispatch.cpp

```diff
@@ -0,0 +1,69 @@
+#include "aten_dispatch.h"
+#include "torch/csrc/jit/interned_strings.h"
+#include "torch/csrc/utils/functional.h"
+
+#include <unordered_map>
+
+// ${generated_comment}
+
+namespace torch { namespace jit {
+
+using autograd::Variable;
+using autograd::variable_list;
+using at::Scalar;
+using at::Tensor;
+using at::IntList;
+using at::TensorList;
+using operator_constructor = std::function<TensorOp(jit::Node*)>;
+
+namespace {
+
+variable_list pack_list(Tensor v) { return { std::move(v) }; }
+variable_list pack_list(Scalar v) { return { v.toTensor() }; }
+variable_list pack_list(std::vector<Tensor> t) { return fmap<Variable>(t); }
+variable_list pack_list(std::tuple<Tensor, Tensor> v) {
```

Comment:
You can't do this, it will trigger a Clang bug on OS X. See #3234

Reply:
I've been thinking about it, but I wrote all of this code on OS X and it builds just fine for me.

Reply:
Maybe it was fixed in Apple Clang 9.0

Reply:
I would fix it, but idk how to reproduce and ensure that it works

Reply:
Yes, it is fixed in Clang 9.0. But, as you can see, we have plenty of users on clang-802.0.42 (me included.)  I think in principle, you should be able to install an older version of Xcode and then use xcode-select to put it into your path as the preferred version. Apple LLVM version 8.1.0 (clang-802.0.42) repros the problem.

Reply:
Ok, that's what I needed. I'll downgrade and see